.gitignore

```diff
@@ -1,9 +1,10 @@
 __pycache__/
 *.pyc
 .coverage
+.env
 hc.sqlite
 hc/local_settings.py
 static-collected
-
+hc_venv/
```

Comment:
This does not need to be in the gitignore since different people set up their virtualenvs differently. A better workaround would be to have the virtualenv ouside the project directory or better still use virtualenv wrappers. 
http://virtualenvwrapper.readthedocs.io/en/latest/install.html

Reply:
I have removed it. Will read on virtualenv wrappers for future use.